components/checkbox/index.css

```diff
@@ -150,6 +150,12 @@ governing permissions and limitations under the License.
     }
   }
 
+  &:not(.is-readOnly):active {
```

Comment:
This approach will not work in case the SWC wants to disable downstate for checkbox. This is tightly coupled with the css transform. We need to cascade the `--spectrum-downstate-width` and `--spectrum-downstate-height` to this css.

Reply:
Hey Rajdeep! Can you help me understand what needs to change? One piece of context that might be helpful is for components like Checkbox which are smaller than 24px, the transform perspective is not based on the width and height of the component, it is based on a set value [per the design spec](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=9069-310&t=CQ2ikZHR2mb5Ku6g-0).